### PR TITLE
Add guidance page for Organisation Permissions

### PR DESCRIPTION
**Pull request by steventux**
## Context

The Permissions model, and the interplay betweeen user and organisation permissions is confusing. 
Until we have the capacity to improve the overall design, we'd like to add some guidance.

## Changes proposed in this pull request

Add org permissions guidance page
Link to org permissions guidance page from service guidance page

image

## Link to Trello card

https://trello.com/c/okzVrRPW/33-add-guidance-to-help-users-understand-how-permissions-work

## Things to check

- [x] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [x] API release notes have been updated if necessary
- [x] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/controllers/provider_interface/content_controller.rb [hunk 1]
```diff
@@ -53,5 +53,9 @@ def dates_and_deadlines
     def roadmap
       render_content_page :roadmap
     end
+
+    def organisation_permissions
+      render_content_page :organisation_permissions
```
Comment (JR-G) [inline app/controllers/provider_interface/content_controller.rb:58]: Does this page need breadcrumbs, similar to the dates and deadline one?
image

Comment (JR-G) [inline app/controllers/provider_interface/content_controller.rb:58]: Does this page need breadcrumbs, similar to the dates and deadline one?
image

Comment (steventux) [reply app/controllers/provider_interface/content_controller.rb:58]: :+1: Good catch yes probably does though not mentioned in trello.

Comment (steventux) [reply app/controllers/provider_interface/content_controller.rb]: @JR-G https://github.com/DFE-Digital/apply-for-teacher-training/pull/7011/commits/91244c17a68a2bf80ba8c8e16fb0730d63d21a68

**Review by JR-G — APPROVED** [review]
thanks steve

**Pull request merged**